frontends/election-manager/src/screens/manual_data_import_precinct_screen.tsx

```diff
@@ -404,22 +721,33 @@ export function ManualDataImportPrecinctScreen(): JSX.Element {
                         </ContestDataRow>
                       )
                     )}
+                  {contest.type === 'candidate' && contest.allowWriteIns && (
+                    <AddWriteInRow
+                      addWriteInCandidate={(name) =>
+                        addWriteInCandidate(contest.id, name)
```

Comment:
so what is the candidate ID we are giving these write-ins? It should still probably start with __write-in or something right, so we know its a write-in?